scripts/package.json

```diff
@@ -7,9 +7,13 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
+        "chalk": "^5.0.0",
+        "fs-extra": "^10.0.1",
+        "minimatch": "^3.0.4",
         "prettier": "2.5.1",
-        "yargs": "17.3.1",
-        "minimatch": "^3.0.4"
+        "prompt-sync": "^4.2.0",
+        "sort-package-json": "^1.54.0",
```

Comment:
We probably can get rid of this `sort-package-json` dependency if it is not used.

Reply:
used now